simphony/core/material_relation_definitions.yml

```diff
@@ -3,6 +3,7 @@
   class_name : LennardJones
   allowed_number_materials: [1, 2]
   doc_description: Approximated potential energy of interaction between a pair of neutral atoms or molecules based on their distance of separation.
+  domain: [MD]
```

Comment:
@ahashibon,  I added a "domain" so we could label the different material relations.  Is that okay? Is the name okay?  

Currently it is just used in the documentation (for example, an extra column is added to the a table)